### PR TITLE
Create _child_terra_billing_project_disable.Rmd

### PR DESCRIPTION
**Pull request by KatherineCox**
(in progress)

adds instructions on how to "disable" a Terra Billing Project (i.e. remove Google Billing Account)

TODO:
- [ ] point to main branch for child doc before merging

### CODE REVIEW

File: child/_child_terra_billing_project_disable.Rmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ottrpal::include_slide("https://docs.google.com/presentation/d/1ib--pXZdu-n-c3b28n73SILA0SrIF5WhdMMaae3DTEI/edit#slide=id.g21148e49334_0_18")
+    ```
+
+1. The page should now indicate that there is no linked billing account.
+
+    ```{r, echo=FALSE, out.width = '100%', fig.alt='Screenshot of a Terra Billing Project management page.  The Billing Account information which says "No linked billing account" is highlighted.'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1ib--pXZdu-n-c3b28n73SILA0SrIF5WhdMMaae3DTEI/edit#slide=id.g21148e49334_0_24")
+    ```
+
+If necessary, you can restore funding to the Billing Project and associated Workspaces by clicking the teardrop icon and selecting "Change Billing Account".  However, Google makes no promises about how long the Workspace contents will remain available after you disable funding, so it is best not to rely on them.
```
Comment (avahoffman) [inline child/_child_terra_billing_project_disable.Rmd:74]: Did you mean for this last section to be numbered? It's currently not, meaning it has a different indentation than the rest. Just making sure this is the intent :)

**Review by avahoffman — APPROVED** [review]
Looks great!

**Review by ehumph — APPROVED** [review]
lgtm!

File: child/_child_terra_billing_project_disable.Rmd (hunk 1 above)
Comment (KatherineCox) [reply child/_child_terra_billing_project_disable.Rmd:74]: Yeah, I left it unnumbered since it's not part of the process of disabling.

**Pull request merged**